plotly/tools.py

```diff
@@ -1647,6 +1699,193 @@ def create_streamline(x, y, u, v,
+
+        # Plot
+        fig=Figure()
+        fig['data'].append(ohlc_increase)
+        url = py.plot(fig, filename='ohlc_increase')
+        ```
+
+        Example 2: Plot ohlc chart increasing and decreasing periods
+        ```
+        import plotly.plotly as py
+        import plotly.tools as tls
+        from plotly.graph_objs import *
+
+        # Add data
+        open_data = [33.0, 33.3, 33.5, 33.0, 34.1]
+        high_data = [33.1, 33.3, 33.6, 33.2, 34.8]
+        low_data = [32.7, 32.7, 32.8, 32.6, 32.8]
+        close_data = [33.0, 32.9, 33.3, 33.1, 33.1]
+
+        # Create ohlc increasing periods
+        ohlc_increase = tls.TraceFactory.create_ohlc_increase(open_data,
+                                                              high_data,
+                                                              low_data,
+                                                              close_data)
+
+        # Create ohlc decreasing periods
+        ohlc_decrease = tls.TraceFactory.create_ohlc_decrease(open_data,
+                                                              high_data,
+                                                              low_data,
+                                                              close_data)
+
+        # Plot
+        fig=Figure()
+        fig['data'].append(ohlc_increase)
+        fig['data'].append(ohlc_decrease)
+        url = py.plot(fig, filename='ohlc_increase_decrease')
+        ```
+
+        Example 3: Plot ohlc chart with date labels
+        ```
+        import plotly.plotly as py
+        import plotly.tools as tls
+        from plotly.graph_objs import *
+
+        # Add data
+        open_data = [33.0, 33.3, 33.5, 33.0, 34.1]
+        high_data = [33.1, 33.3, 33.6, 33.2, 34.8]
+        low_data = [32.7, 32.7, 32.8, 32.6, 32.8]
+        close_data = [33.0, 32.9, 33.3, 33.1, 33.1]
+        dates = ['3/09', '6/09', '9/09', '12/09', '3/10']
+
+        # Create ohlc increasing periods
+        ohlc_increase = tls.TraceFactory.create_ohlc_increase(open_data,
+                                                              high_data,
+                                                              low_data,
+                                                              close_data)
+
+        # Create ohlc decreasing periods
+        ohlc_decrease = tls.TraceFactory.create_ohlc_decrease(open_data,
+                                                              high_data,
+                                                              low_data,
+                                                              close_data)
+
+        # Create layout with dates as x-axis labels
+        fig = dict(data=[ohlc_incr, ohlc_decr],
+              layout=dict(xaxis = dict(ticktext = dates,
+                                       tickvals = [1, 2, 3, 4, 5 ])))
+
+        # Plot
+        url = py.plot(fig, filename='ohlcs_dates',
+                      overwrite=True, validate=False)
```

Comment:
:cow2: `overwrite=True`? Can this just be:

`url = py.plot(fig, filename='ohlcs_dates', validate=False)`